meshpy/header_functions.py

```diff
@@ -425,6 +428,19 @@ def set_header_static(
         )
     )
 
+    given_time_arguments = [arg for arg in (time_step,n_steps,total_time) if arg is not None]
+    if len(given_time_arguments) < 2:
+        raise ValueError('At least two of the following arguments "time_step", "n_steps" or "total_time" are required')
+
+
+    if time_step is None:
+        time_step = total_time/n_steps
+    elif n_steps is None:
+        n_steps = total_time/time_step
```

Comment:
Should this be casted to an int?

```suggestion
        n_steps = int(total_time/time_step)
```

Reply:
Good catch, I think `round` makes the most sense here.